docs/meshstack.identity-federation.md

```diff
@@ -126,7 +126,7 @@ meshStack supports the following identity provisioning strategies:
 | :-----------: | :-------------------------------------------------------------------: | :------------------------------------------------------------------------: |
 |   OpenStack   |         [deprecated](./meshstack.openstack.index.md#meshidp)          |         [supported](./meshstack.openstack.index.md#enterprise-idp)         |
 | Cloud Foundry |    [supported](meshstack.cloudfoundry.index.md#uaa-configuration)     |                                 _planned_                                  |
-|  Kubernetes   | [supported](./meshstack.kubernetes.index.md#api-server-configuration) |                                 _planned_                                  |
+|  Kubernetes   | [deprecated](./meshstack.kubernetes.index.md#api-server-configuration) |                                 _planned_                                  |
```

Comment:
note: externally provisioned is supported (this is missing!)

I think I made that change in https://github.com/meshcloud/meshcloud-docs/pull/646/files